<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parking_personnels</class>
 <widget class="QDialog" name="parking_personnels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>711</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="ajout">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="supp">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>430</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="mod">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>415</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="aff">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>430</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_aj">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>375</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_supp">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>375</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_mod">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>405</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_aff">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>425</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>590</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>590</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic0">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>935</width>
       <height>725</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_pic0</zorder>
    <zorder>label_mod</zorder>
    <zorder>label_aff</zorder>
    <zorder>label_aj</zorder>
    <zorder>label_supp</zorder>
    <zorder>supp</zorder>
    <zorder>ajout</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton</zorder>
    <zorder>mod</zorder>
    <zorder>aff</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>514</x>
       <y>430</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>640</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_add">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>410</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_retour">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>630</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic1">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>981</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>label_pic1</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>lineEdit_id</zorder>
    <zorder>lineEdit_nom</zorder>
    <zorder>lineEdit_prenom</zorder>
    <zorder>label_add</zorder>
    <zorder>ajouter</zorder>
    <zorder>label_retour</zorder>
    <zorder>pushButton_2</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_ids">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ids">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>429</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>610</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_supp2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>370</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic2">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>-11</y>
       <width>981</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_retour1">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>610</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>320</width>
       <height>510</height>
      </rect>
     </property>
    </widget>
    <zorder>label_pic2</zorder>
    <zorder>label_ids</zorder>
    <zorder>lineEdit_ids</zorder>
    <zorder>label_supp2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label_retour1</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>tableView</zorder>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>120</y>
       <width>320</width>
       <height>510</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>360</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>440</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>610</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>260</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>360</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PRENOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_retour2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>610</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mod2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>430</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_pic3</zorder>
    <zorder>tableView_3</zorder>
    <zorder>lineEdit_7</zorder>
    <zorder>lineEdit_6</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_8</zorder>
    <zorder>label_retour2</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>label_mod2</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prenom</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>351</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_retour3">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>630</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>630</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>label_pic4</zorder>
    <zorder>comboBox</zorder>
    <zorder>lineEdit</zorder>
    <zorder>tableView_2</zorder>
    <zorder>label_retour3</zorder>
    <zorder>pushButton_7</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
